client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -36,6 +39,48 @@ export default React.createClass( {
 		this._pressableStore = pressableStoreStep;
 	},
 
+	scrollUp() {
+		// Didn't use setInterval in order to fix delayed scroll
+		while ( window.pageYOffset > 0 ) {
+			window.scrollBy( 0, -10 );
+		}
```

Comment:
@aduth + @ehg in places like these where we actually _need_ to access `window`, should we consider safeguarding the variable for tests' sake? like this…

```js
const window = ( 'undefined' === window )
	? { scrollBy: noop }
	: window;
```

or do we have a standard `window` "mock" for these cases which does that automatically? would we benefit from having…

```
import { window } from 'lib/bom';
```